src/mrsimulator/simulator/__init__.py

```diff
@@ -419,9 +419,11 @@ def run(
                 backend="loky",
             )(jobs)
 
-            gyromagnetic_ratio = method.channels[0].gyromagnetic_ratio
             B0 = method.spectral_dimensions[0].events[0].magnetic_flux_density
-            w_ref = np.abs(B0 * gyromagnetic_ratio * 1e6)
+            # gyromagnetic_ratio = method.channels[0].gyromagnetic_ratio
+            # w_ref = np.abs(B0 * gyromagnetic_ratio * 1e6)
+            isotope = Isotope(symbol=method.channels[0].symbol)
```

Comment:
`method.channels[0]` is already a Isotope instance. No need to create another. could just use
```py
isotope = method.channels[0]
```